### PR TITLE
Feature/tasks page

### LINKED ISSUE #6: Frontend for Task Page - Anoushka
**Please describe the feature you are implementing**
- Implementing Task Page for Week 5 sprint.
- Need headings (title and subtitle for tasks), Start button, Question Mark button

**Additional context**
-  Design that needs to be implemented:
Screen Shot 2021-01-31 at 3 22 00 PM

### LINKED ISSUE #7: Frontend for Task Page - Ryan
**Please describe the feature you are implementing**
- Implementing the task list component for the frontend task page.

**Additional context**
Screen Shot 2021-01-31 at 3 22 00 PM

### PR DESCRIPTION
**Pull request by anoushkadave**
Completed frontend for Tasks (Landing) Page. All frontend is working except for media queries to support different browsers.

Please delete this branch after PR is approved.

close #6 
close #7

### CODE REVIEW

**Review by thedestroyer4312 — APPROVED** [review]
Looks good to me, Anoushka!

File: src/pages/tasks.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <title>Document</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <!--External Stylesheet-->
+        <link href="../styles/tasks.css" rel="stylesheet">
+
+        <!--Comfortaa Font-->
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
```
Comment (TheHiddenLayer) [inline src/pages/tasks.html:13]: FYI, we'll be taking this out before we merge `base: master <-- branch: styles`

File: src/styles/tasks.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
+* {
+  font-family: "Comfortaa", sans-serif;
+}
+
+body {
+  background-color: #ff6767;
```
Comment (TheHiddenLayer) [inline src/styles/tasks.css:6]: I've also added global background colors, so fyi this will also be adjusted!

**Review by ryanbuui — APPROVED** [review]
LGTM

File: src/styles/tasks.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+.task {
+  height: 3.5em;
+  width: 25em;
+}
+
+.task-time {
+  height: 3.5em;
+  width: 5em;
```
Comment (TheHiddenLayer) [inline src/styles/tasks.css:105]: nice, thanks for not hardcoding.

**Review by TheHiddenLayer — APPROVED** [review]
Approved

**Pull request merged**